src/components/Button/Button.styles.js

```diff
@@ -10,8 +10,11 @@ const styles = StyleSheet.create({
   },
   containerSecondary: {
     // Make pressable area bigger, take into account for spacing
+    backgroundColor: WHITE,
+    height: 54,
+    borderRadius: 54 / 2,
```

Comment:
para que fue necesario esto?

Reply:
noté que con estos cambios perjudicaba el _Ver mas_ por ejemplo, lo que hice ahora fue setearlos en el ModalCard.styles, removí las props _backgroundColor_ y _borderRadius_ ya que no eran necesarias.